### ISSUE TITLE
Set up projects to track progress and tasks for Reactome2GO

### ISSUE THREAD

**Issue by ukemi** [geneontology/pathways2GO#167]
It is time to move from the old notes in the Google Drive to a more formal structure. We will set up projects for each of the main objectives in the Reactome2GO conversion and add tasks to those projects. The ultimate goal is to achieve a list of specific tasks that can be enumerated and scoped the way the Lori did for the MGI imports.

**Comment by ukemi** [comment]
There are currently two project in the pathways2GO repository. Should we try to roll them into the top-level project? copy them to the top level?

**Comment by deustp01** [comment]
How are they related to the straw man framework?

**Comment by deustp01** [comment]
A quick sanity check on the whole structure on Monday will probably be a good idea.

**Comment by ukemi** [comment]
I'll add it to the call agenda.

**Comment by ukemi** [comment]
Progress so far. I have created top level GOC projects for each of the projects we discussed in the straw-man proposal. I have added tickets to those projects. I have gone to the Pathways2GO repo and assigned each ticket to a project. I have also added general labels to the tickets to give an overall view of who's court the ticket is in. On Monday, let's look at the columns in each project and refine them with respect to proposed workflow. Then I think we can close out this ticket.
